ctrd/utils.go

```diff
@@ -65,13 +66,75 @@ func resolver(authConfig *types.AuthConfig, resolverOpt docker.ResolverOptions)
 	return docker.NewResolver(options), nil
 }
 
+//GetWeightDevice Convert weight device from []*types.WeightDevice to []specs.LinuxWeightDevice
```

Comment:
leave a space between `//` and the function name may be better?

Reply:
yep, you are right